octoprint_mrbeam/static/js/mother_viewmodel.js

```diff
@@ -401,14 +407,35 @@ $(function () {
             }
         };
 
+        self.state.isPaused.subscribe(function (newIsPaused) {
```

Comment:
just FYI:
you could also implement a method `self.onPrintPaused` which would have the same effect.